doc/lua_api.txt

```diff
@@ -2551,12 +2551,14 @@ and `minetest.auth_reload` call the authetification handler.
 * `minetest.delete_area(pos1, pos2)`
     * delete all mapblocks in the area from pos1 to pos2, inclusive
 * `minetest.line_of_sight(pos1, pos2, stepsize)`: returns `boolean, pos`
-    * Check if there is a direct line of sight between `pos1` and `pos2`
+    * Checks if there is anything other than air between pos1 and pos2.
+    * Returns false if something is blocking the sight.
     * Returns the position of the blocking node when `false`
     * `pos1`: First position
     * `pos2`: Second position
-    * `stepsize`: smaller gives more accurate results but requires more computing
-      time. Default is `1`.
+    * `stepsize`: Kept for compatibility, the starting point of the
```

Comment:
Maybe

> `initial_jump`: The starting point of the line is translated towards the end by initial_jump, defaults to 1. Set to 0 for accurate results. Kept for compatibility reasons to avoid breakage in code written before stepsize was deprecated

Reply:
Yes better, if this must be used at all, i can't see any need to use it though.